Card/src/main/java/com/paypal/android/card/CardAPI.kt

```diff
@@ -15,43 +18,54 @@ internal class CardAPI(
         val apiRequest = requestFactory.createConfirmPaymentSourceRequest(orderID, card)
         val httpResponse = api.send(apiRequest)
         val correlationId = httpResponse.headers["Paypal-Debug-Id"]
-        return if (httpResponse.status == HTTP_OK) {
-            runCatching {
-                val json = PaymentsJSON(httpResponse.body)
-                val status = json.getString("status")
-                val id = json.getString("id")
+        if (URLUtil.isValidUrl(apiRequest.path).not()) {
```

Comment:
Would using `!` instead of `.not()` be clearer here?

Reply:
Sounds good!